tests/src/test/scala/whisk/core/connector/test/MessageFeedTests.scala

```diff
@@ -1,11 +1,12 @@
 /*
- * Copyright 2015-2016 IBM Corporation
```

Comment:
interesting - how did this not get caught by scanCode?

Reply:
Not sure. Maybe @mrutkows knows?

Reply:
Because scan code default config needs to be updated to remove IBM license as a valid license

Reply:
Also sounds like the test directory isn't scanned. https://github.com/apache/incubator-openwhisk-cli/issues/222#issuecomment-373798479